### PR TITLE
Add materialized "approved_at" field to approvals

### PR DESCRIPTION
**Pull request by cmc333333**
Split from #454, this creates an `approved_at` field for approvals. The field is set when the approval enters the approved state. Also backfills existing data.

Also simplifies options on the approval factory (rather than setting non-useful defaults).

### CODE REVIEW

File: spec/models/approval_spec.rb [hunk 1]
```diff
@@ -20,4 +20,21 @@
...
+
+    it 'is nil when actionable' do
+      approval.make_actionable!
+      expect(approval.approved_at).to be_nil
+    end
+
+    it 'is set when approved' do
+      approval.make_actionable!
+      approval.approve!
+      expect(approval.approved_at).not_to be_nil
```
Comment (afeld) [inline spec/models/approval_spec.rb]: Might want to check that it's actually persisted.

**Comment by afeld** [comment]
:shipit: 

(needs merge from `master`)

File: spec/models/approval_spec.rb (hunk 1 above)
Comment (cmc333333) [inline spec/models/approval_spec.rb]: Can do

**Comment by cmc333333** [comment]
Updated w/ master and persistence test

**Pull request merged**